src/styles/globals.scss

```diff
@@ -47,3 +47,79 @@ input {
   width: 0;
   height: 0;
 }
+
+.navMenu {
```

Comment:
can we move this into `src/components/NavListItem/NavListItem.module.scss`?

you'll need to rename the top level className so that it gets imported in as a css module

Reply:
yes, yes. i'll move it.